common/ta_errors.h

```diff
@@ -147,6 +147,10 @@ typedef enum {
   /**< Failed to created/get chid or epid or msg_id in MAM */
   SC_MAM_FAILED_WRITE_HEADER = 0x0B | SC_MODULE_MAM | SC_SEVERITY_FATAL,
   /**< Failed to write header in MAM */
+  SC_MAM_MESSAGE_NOT_FOUND = 0x0C | SC_MODULE_MAM | SC_SEVERITY_FATAL,
```

Comment:
Where do we use this error code?

Reply:
they are used in `ta_mam_api_bundle_read()`